pkg/leader/leader.go

```diff
@@ -54,7 +55,7 @@ const PodNameEnv = "POD_NAME"
 func Become(ctx context.Context, lockName string) error {
 	log.Info("Trying to become the leader.")
 
-	ns, err := myNS()
+	ns, err := k8sutil.GetOperatorNamespace()
```

Comment:
We can also remove the function myNs() as it is not being used anymore.

Reply:
Done, my bad, I should have done it with previous commit. Sorry for that "one more" iteration.